README.md

```diff
@@ -94,13 +89,12 @@ does. In most cases, it should be compatible with your existing crontab.
 
 There are, however, a few exceptions:
 
-- First, Supercronic supports second-resolution schedules: under the hood,
+- First, Supercronic supports second-resolution schedules: Under the hood,
```

Comment:
Style guides vary on whether to use uppercase after a colon, but when the components can be broken into complete sentences, I think it reads better. Flame wars have been started over less, though. 🔥